Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteLayerTreeHostIOS.mm

```diff
@@ -92,6 +92,7 @@
 
 #if ENABLE(MODEL_ELEMENT)
     case PlatformCALayer::LayerTypeModelLayer:
+        RELEASE_ASSERT_WITH_SECURITY_IMPLICATION(m_drawingArea->page().preferences().modelElementEnabled());
```

Comment:
Seems like this is racy, since you could have a model layer coming in flight from the web process, and then change the prefs in the UIProcess before this is called.

Reply:
Hmm, that is true, I suppose. Do you think we should just make a empty view instead?

Or kill the WP, in MESSAGE_CHECK style?